tests/test_github_processor.py

```diff
@@ -80,10 +85,46 @@ def test_add_github_resource_processor(self, mock_request):
                 'repository': 'my-repository',
                 'watchers': 4,
                 'stars': 1,
+                'forks': 10,
                 'source': 'github',
-                'date': datetime.date.today()
+                'date': datetime.date.today(),
```

Comment:
Using dates without mocking means that the test can fail depending on when they're run. This is unlikely here, as you're using just the day, but fixing it is so easy with https://github.com/spulec/freezegun that we might just do it. I haven't checked if other tests also have this issue, although they potentially do as they use `datetime` as well.